### PR TITLE
Shellcheck

### PR DESCRIPTION
**Pull request by BarbUk**
Fix several shellcheck warning and error

### CODE REVIEW

**Comment by BarbUk** [comment]
The resulting script was tested on debian 9 with lxc 2

File: check_lxc.sh [hunk 1]
```diff
@@ -51,19 +51,20 @@ sleep=5                 # define the sleep time between cpu checks
...
 done
 ################################################################################
 # LXC 0.x and 1.x commands are different. The following lxc commands are required.
-lxcversion=$((lxc-version 2>/dev/null || lxc-start --version) | sed 's/.* //' | awk -F. '{print $1}')
+lxcversion=$( (lxc-version 2>/dev/null || lxc-start --version) | sed 's/.* //' | awk -F. '{print $1}' )
 
 if [[ $lxcversion -eq 0 ]]; then
-  for cmd in lxc-list; 
-  do if ! `which ${cmd} 1>/dev/null`
+  # shellcheck disable=2043
```
Comment (Napsty) [inline check_lxc.sh:68]: What do you mean by that?

**Comment by Napsty** [comment]
Thanks @BarbUk 
These are a lot of speed improvements, learning a lot here ;-)
Would you be so kind and adapt the PR for the latest release (currently 0.6.2) to be able to merge? 
Thx

**Comment by BarbUk** [comment]
Hey, thanks for the feedback.
I've merged the new version into my branch and resolved the conflicts.

I've added some new fix from shellcheck.
The resulting script has been tested against lxc 2.0.7-2 on a debian stretch.

**Comment by Napsty** [comment]
Successfully tested on 

- Ubuntu 14.04 with LXC 1.0.8
- Ubuntu 18.04 with LXC 3.0.2 (however: #9 )

Thx for the contribution!

**Pull request merged**